### ISSUE TITLE
Make sure presentations build so we can get started on them

### ISSUE THREAD

**Issue by cgrandin** [cgrandin/hake-assessment#406]

**Comment by andrew-edwards** [comment]
I did check that the beamer/SRG/folders we have match the talks listed on the SRG agenda. And Stacey said the screen will be 16:9, so we can probably utilise that.

**Comment by taylori** [comment]
Either before or after they can build again, perhaps we could divide up the presentations so to spread the burden of updating them.

**Comment by aaronmberger** [comment]
We usually, and I'm guessing will still will, have the standard 4:
1. Data
2. Assessment
3. Sensitivities and Retrospectives
4. Management

I'm fine to do any of them, but don't think I've ever done 1 or 4.  Any takers on 1-4?

**Comment by andrew-edwards** [comment]
I can do Assessment this time maybe. Think I did 3 last time. Haven't looked at it yet though. Makes sense that we all do one again.

**Comment by cgrandin** [comment]
I could do 3, I've done mostly 1 in the past. I'm just wondering how it's going to work doing the sensitivity presentation before the assessment one, like has been planned.

**Comment by cgrandin** [comment]
Assessment presentation builds as of commit dac8a9dbcd88c6e4beb2c39f2190e27db91611a6

**Comment by cgrandin** [comment]
Management presentation builds as of commit dac8a9dbcd88c6e4beb2c39f2190e27db91611a6

**Comment by aaronmberger** [comment]
Sensitivities are after the stock assessment methods/results on the current schedule.  In fact, they are the next day.  I mentioned to Michelle that ideally those two would go back to back as much discussion on the assessment will inevitable be related to sensitivities.

I can try the data one this year.

**Comment by taylori** [comment]
Sounds like the plan is coming together. If we have Andy do 2=Assessment, then that leaves 4=Management for me. We can also tag team a little bit as needed if there are particular areas of any given talk that would benefit from an additional person speaking to any issue.

**Comment by aaronmberger** [comment]
Great, I'll be starting to work on the data presentation today and hopefully finish by tomorrow.

**Comment by cgrandin** [comment]
Sensitivity presentation builds as of commit eb1613acc81a86b9bfa368240aa6a63718f48811.
I'll start working on it now.

**Comment by andrew-edwards** [comment]
Presumably we're keeping the structures pretty similar to before. For instance there are bridging plots in the main Assessment talk. I guess if anyone wants to add something into their talk that may appear in another one they should make an Issue first just to check. I think structures before worked fine, so shouldn't be a problem. I'll put in a few slides about Phi, since it's going to come up.

**Comment by taylori** [comment]
Once the presentations are all minimally updated we could each build all of them to look them over and then have a discussion about whether the structure and flow looks good. We could also have a phone call to discuss on Friday if that would be useful, or just rely on issues to sort stuff out.

**Comment by aaronmberger** [comment]
I guess I vote for a phone call on Friday as we aren't meeting before the SRG meeting, but could go either way.  I'm available on Friday anytime other than 10-12.

**Comment by andrew-edwards** [comment]
I could do a short call - probably worth it. I didn't get very far yesterday (catching up on numerous emails) and didn't even push, but will crack on today.

**Comment by andrew-edwards** [comment]
Maybe people can check off here when they want someone to look through their talk (i.e. it's finished enough).

- [ ]  Data (Aaron)
- [ ]  Assessment (Andy)
- [ ]  Sensitivities and retrospectives (Chris)
- [ ]  Management (Ian)

And, yes, I saw someone have such a check list yesterday and partly just wanted to see how well it worked!

**Comment by aaronmberger** [comment]
Can you see me check by the data one?  I just checked it and refreshed...

**Comment by taylori** [comment]
Yes, check works. The management one could use some refining on some of the bulleted points at the end among other places, but it good enough to look over.

**Comment by cgrandin** [comment]
I can do a phone call tomorrow, anytime. I'm still working on the Sensitivities presentation, although I had a 3-hour meeting this morning which caused a bit of delay.

**Comment by cgrandin** [comment]
Sensitivity one is close, any comments would be appreciated. Cheers

**Comment by cgrandin** [comment]
Aaron, I just looked over the Data presentation, and only have to questions:

- Was tribal catch 0 t?
- Should we remove the slide on Acoustic survey extrapolated/unextrapolated?

**Comment by aaronmberger** [comment]
I think your right about the extrapolation figure as the SRG has agreed to that, so I'll change to the main survey figure (Figure 9).

Tribal catch 0 t?  Yes and no.  The tribal catch is often times lumped in with shoreside catch, depending on how fish tickets are coded at dockside/processing.  I think what I'll do is make the tribal a sub-bullet of the shoreside and list the 2017 tribal catch there.

**Comment by aaronmberger** [comment]
The tribal issue is similar to the research catch issue in that only some specifically coded 'research' catch gets labelled as research in the database.

**Comment by taylori** [comment]
Just built and looked over data presentation. 
It looks really good to me, but here are a few minor suggestions that you can feel free to ignore.

Slide 1:
- "some Shoreside catch in April" should maybe clarify that this is in Canada (U.S. start date is May 15, right?)
- what about ""Highest coastwide catch ever" instead of "Highest coastwide catch in time-series"?

Slide 6:
- I like your plan for sub-bullet for Tribal catch
- I wonder if it wouldn't be better to remove the bullets for "Research" as well, since this is really just specific to landed research catch caught under an exempted fishing permit and does not include any survey catch, for instance.

Slide 8: Maybe change the slide title to "Cumulative catch by month" since x-axes don't have labels?

Slide 11: I'm getting an error:
```
## Error in plotBubbles(t(d), dnam = TRUE, bg = bg.col, las = 1, powr = 0.5,
: could not find function "plotBubbles"
```

Slide 17: Maybe change slide title from "Maturity North and South of 34.5◦ latitude" to "Updated maturity and fecundity estimates"

Slide 18: Change "SS3" to "SS" or "Stock Synthesis". Rick has been trying (not totally successfully) to convince people that the name was never SS3).

Slide 19: Maybe change "WCGOP data (bycatch)" to "bycatch in non-target fisheries" to cover any discard data from fisheries that aren't part of the WCGOP observer program, such as anything in Canada.

**Comment by taylori** [comment]
I see that plotBubbles identified in the error on slide 11 is in PBSmodelling, which we have a require for in all.r. I'm not sure why it wasn't working, but I'll look into it.

**Comment by aaronmberger** [comment]
Mine was working fine, but I'm rebuilding the presentation now.  I agree with all your suggestions Ian and will make changes accordingly.  Regarding the "Shoreside catch in April" - a bit tricky in the details, but in general yes it is mostly Canada b/c US doesn't start until May 15, but there is very minor levels of US shoreside catch (<15 t /month) in the database for months prior to May (presumably bycatch or from tribal catches...?).  It is so small that is probably more clear-cut to say Canadian Shoreside catch starts in April....

**Comment by taylori** [comment]
Turns out I've never been able to load PBSmodeling due to this error:
```
library(PBSmodelling)
Error: package or namespace load failed for 'PBSmodelling':
 package 'tcltk' does not have a namespace
```
The internet has lots of stuff about problems installing tcltk on Macs, but not so much about Windows computers. Since it doesn't seem to be a problem for anything else, I won't worry about it for now.

**Comment by taylori** [comment]
@aaronmberger regarding the U.S. Shoreside catch prior to May 15, I took a quick look at the ~200 records from our PacFIN catch extraction
pcatch[pcatch$YEAR==2017 & pcatch$TDATE < as.POSIXct("2017-05-15"),]
and it appears that they are mostly associated with DAHL_SECTOR = 4, which is "Shoreside Nonwhiting Trawl Sector" (seems to be any IFQ trawl landing which is less than 50% hake), so presumably just bycatch in other fisheries that didn't get discarded prior to landing. There were only 3 tows with amounts greater than 1 ton, which happened to all be in March.

I'm satisfied and don't see any need for change as long as you're happy answering any questions that come up.

**Comment by aaronmberger** [comment]
Thanks for checking on that Ian.  I'm satisfied too.

**Comment by aaronmberger** [comment]
I guess I'm feeling pretty good about our slides so don't see a particular need to connect on a call tomorrow.  Final decision on that....? 

I will have a closer look at the management one tonight and the other 2 tomorrow.

**Comment by taylori** [comment]
@cgrandin here are a few comments on the sensitivity presentation.
Looks really good to me.

Here are two minor things:

Slides 12 and 13 have redundant runs. Would it make sense to remove the "Phi t.v. selectivity" runs from 12 so they only appear on 13?

On slide 14 I was surprised by how little difference in Recruitment in 2014 from the base model there was for the Phi = 0.70 and 2.10, with Phi = 0.21 very different. I wonder if it's worth also including something (perhaps on the same slide) about Recruitment in 2016. Just to see the comparison, I changed 2014 to 2016 in that slide and got the following figure where the differences are pretty striking (and supportive of the higher Phi  values, either 1.4 in the base model or 2.1 in the sensitivity):

![image](https://user-images.githubusercontent.com/4992918/36571570-34caec26-17ec-11e8-9eec-f245a94f36df.png)

**Comment by taylori** [comment]
Forgot to comment on Aaron's question about a phone call.
I would say that unless Andy want's one to discuss anything related to the Assessment presentation, we're probably good without it.

**Comment by andrew-edwards** [comment]
I'm good without a call. Discussing Issues works well.

**Comment by andrew-edwards** [comment]
Aaron - data presentation looks great. I thought you'd cover the new maturity ogive - gave me an easy commit e5ab758 to remove a placeholder.

One thing - should title of 'Weight-at-age by Year and Cohort" slide just be "Weight-at-age by Year"? It's not really tracking cohorts.

Or maybe "Mean Weight-at-age by Year" to match the axis.

I should get the assessment presentation finished up tomorrow - figures are updated, I just have to finish updating some of the text.

**Comment by cgrandin** [comment]
Thanks for the tips Ian.
After looking at the differences in the end-year biomass for the semi-parametric sensitivities, do you think it's useful to look at the recruitment densities for those runs as well? Here's what they look like:
![image](https://user-images.githubusercontent.com/4551051/36607774-7f526dce-187c-11e8-8788-f5da77eff801.png)
![image](https://user-images.githubusercontent.com/4551051/36607786-89b642a4-187c-11e8-98b3-36843e6b2add.png)

**Comment by taylori** [comment]
Good question. Maybe so.

Last year we made a range of arguments of why we needed to increase the flexibility of the time-varying fishery recruitment, but I thought that the strongest one was that without extra flexibility, the estimated 2014 Recruitment was implausible. Strangely, that was also the one that got the most pushback, I thought, where the SRG said that we just changed the model because we didn't like the answer. Now that we have 1 more year of data supporting 2014 as lower than 2010 rather than 3 times higher, I think our choice was clearly the right one.

One of the benefits of the semi-parametric selectivity is that the Laplace approximation method seems to work better for tuning the variance parameter (sigma_s, which plays a similar role to Phi in the standard parameterization). However, the 0.695 that results from that method seems to produce pretty big 2014 and 2016 recruitments. Given the background of last year, I think we are justified. in choosing to be cautious and stick with the current approach.

**Comment by aaronmberger** [comment]
Regarding your comment Andy: 'One thing - should title of 'Weight-at-age by Year and Cohort" slide just be "Weight-at-age by Year"? It's not really tracking cohorts.'

I agree and will make that change to the title.  It is worth pointing out that you can 'see' cohorts along diagonals, but I'll just point that out rather than having it in the title which can create confusion.

**Comment by aaronmberger** [comment]
Ian, the management presentation looks great.  I had only a couple fleeting (excuse the pun) comments/ideas as I went through it, none of great importance.

Slide 3: 2017 base model now indicates that fishing intensity has never been above target (though real close in 1999).  Maybe simply delete ‘mostly’…

Slide 4: ‘Mean wt-at-age’ in the last year or across all years – it could be interpreted either way – though could just say it

Slide 4: title says Methods but it resides in the Introduction section, maybe just change methods to approach…?

Slide 5: I always thought it would be useful to show where we are in relation to the default harvest policy (i.e. on the slope or plateau).  We clearly are on the plateau now, but interesting during years when on the slope or close to the slope (FSPR = 40% or less) – just a thought for possible inclusion next year (default harvest policy plot with different points representing where we would be at in 2018, 2019, and 2020) – though admittedly the harvest policy is never followed so perhaps its nonsense…)

Slide 6: first two bullet points – specify 2018 level of spawning biomass is from which stable and decreases relate – or again, just say it… 

Slide 9: Fixed to Fix

Slide 17: Is it safe to say that forecasts strongly influenced by size of 2014 and 2016 year classes?

**Comment by andrew-edwards** [comment]
Regarding slide 3 - the 1999 intensity dipped just below target this year, it was just above it in last year's assessment. Explains why 'mostly' was there before...

**Comment by aaronmberger** [comment]
The assessment and sensitivity presentations build for me.  I'll hold off looking at those until you guys have finished (unless you already have).  I do see some ** in the assessment one yet.  For me, the sensitivity presentation built in standard format not 16x9 landscape format (probably just need to add back in the \documentclass[aspectratio=169]{beamer} which is currently commented out.

**Comment by andrew-edwards** [comment]
I haven't finished Assessment one but have just pushed and it should be obvious which slides aren't finished yet (flagged with ** or obvious placeholders). So if you have time to take a look then that'd be great. There are a couple of new slides (currently page 26 - Estimate of 2014 cohort, and currently p29 - Spawning biomass table) to explain a few things that will likely come up if we don't explicitly explain them.

I'm going to keep working to the early evening, taking tomorrow off then will work Sunday to finalise this (if not finished today), practice what I want to say (!) and read through document to get familiar with it again.

**Comment by taylori** [comment]
Thanks for the suggestions on the management talk.
After spending an hour debugging a new \emph{R_0} that needed to be either \emph{R}\subscr{0} or $R_0$, I got things working again and just committed some changes in 97ab480.

Here are bullets on the final slide after revisions to the 2nd and 3rd points. I'm happy to edit further if these don't sound good to anyone.
- Based on the default harvest rule, the estimated median catch limit for 2018 is 725,984 t
(with 95% interval from 270,948 to 1,881,590 t).
- Uncertainty is large, but stock is likely above B40% and fishing intensity likely to have been
below 100%.
- Forecasts strongly influenced by size of large but uncertain 2014 cohort and very uncertain
2016 cohort
- Aging 2010 cohort declining due to fishing and natural mortality contributes to median
spawning biomass declining with catch at 350,000 t and above
- Higher risk of falling below B 40% in 2 years than in last year’s assessment

**Comment by taylori** [comment]
@andrew-edwards, thanks for doing the heavy lifting on the hard presentation.
In general it looks great to me. Here are a few minor edits to consider: 
- To make Recruitment plots consistent in slides 24 and 25, you would need to change `end.yr-1` to `end.yr` in call to `make.recruitment.dev.plot()`
- Slide 8: Change "ADMB 11.2" to "ADMB 11.6" (based on comment at top of SS output files).
- Slide 8: "New parameterization for time-varying selectivity" seems too strong considering that a truly new parameterization was considered. Would it be better to say "Modified parameterization..."
- I like diagonals on Pearson plots, but bubbles are positioned in the middle of each year, so it might be good to shift the lines by 0.5 units to the right. I'll look into what it would take to have r4ss do that--presumably not much. Should we add the lines to Fig 25 of the assessment doc? (I seem to remember a discussion about this previously).
Finally, a few spelling things:
- "s" to "z" in "parameterised", "Analysed", and "characterise" would be consistent with the document, but does Canada side with U.S. or U.K. on these spellings? I care more about consistency than which way we choose
- fix "diagrnose"

Again, nice work. I'm quitting for the day but will check in tomorrow and Sunday to give any feedback you wish.

**Comment by andrew-edwards** [comment]
Thanks - those sound good. 

ADMB version I took from document so have made a note to update it (I'm not touching the document at all now, just noting any edits).

'New parameterization' is indeed a bit strong (I wondered that earlier), and we should change it in one-page summary (as it causes alarm because people think the model is changing). Or remove it from one-page summary as it doesn't really have any bearing on results (first bridging plot).

Diagonals on bubble plots were there automatically - strange, as I tried (and gave up) trying to do them for the document. 

I use -ise etc., so may stick with that for the talk -- the document is consistent with itself though (except one Canadian/UK spelling buried somewhere that I let go because I was tired and it amused me to leave it in!).
